File: beecrowd_ 1117.cpp
#include <stdio.h>
 
int main() {
 
    float nota, media = 0.0;
    int count = 0;
    
    while(count < 2) {
        scanf("%f", &nota);
        if (nota < 0 || nota > 10) printf("nota invalida\n");
        else {
            media += nota;
            ++count;
        }
    }
    printf("media = %.2f\n", media/2.0);
 
    return 0;
}

File: BeeCrowd/Python/beecrowd_1117.py
media = 0.0
count = 0

while(count < 2):
    nota = float(input())
    if (nota < 0 or nota > 10):
        print("nota invalida")
    else:
        media += nota
        count += 1
    
print("media = %.2f" %(media/2.0))
